SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -578,6 +578,17 @@ class CSolver {
    */
   inline virtual void SetPrimitive_Limiter(CGeometry *geometry, const CConfig *config) { }
 
+  /*!
+   * \brief Compute the projection of a variable for MUSCL reconstruction.
```

Comment:
Add a note that the result should be halved when adding to i and subtracting from j.